src/plugins/region_map/public/components/region_map_options.tsx

```diff
@@ -197,7 +201,7 @@ function RegionMapOptions(props: RegionMapOptionsProps) {
 
       <EuiSpacer size="s" />
 
-      <WmsOptions {...props} />
+      <WmsOptions {...props} tmsLayers={tmsLayers} />
```

Comment:
nit: could you please enhance a bit the props here - pass only necessary props [ `stateParams`, `setValue`, `tmsLayers` ]. This will potentially decrease the component re-renders.

Reply:
done